packages/forms/resources/views/components/wizard.blade.php

```diff
@@ -123,6 +123,7 @@
             'border-b border-gray-200 dark:border-white/10' => $isContained,
             'rounded-xl bg-white shadow-sm ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10' => ! $isContained,
         ])
+        @style(['overflow: scroll'])
```

Comment:
Can this be replaced with the `overflow-x-auto` class instead of an inline style?

Reply:
Yes, it works. I've made a commit.